wasm-chain-integration/Cargo.toml

```diff
@@ -25,6 +25,7 @@ thiserror = "1"
 byteorder = "1.4"
 tinyvec = {version = "1.5", features = ["alloc"]}
 slab = "0.4.5"
+ptree = "0.4.0"
```

Comment:
Looking at the dependencies of this crate I do not think we should have it as part of the node.

To achieve this we should  add a new feature to the crate which is off by default.

cargo-concordium should then enable this feature so it can use the tree display functionality.

Reply:
done